front/pages/w/[wId]/builder/data-sources/managed.tsx

```diff
@@ -605,8 +605,9 @@ export default function DataSourcesView({
                 take into account files with more than 750Kb of extracted text.
               </li>
               <li>
-                <span className="font-bold">Github</span>: take into account
-                files with more than 750Kb of extracted text.
+                <span className="font-bold">Github</span>: Dust only gathers
+                data from issues and discussions (not from pull requests, nor
```

Comment:
actually it does index pull requests, but only "as an issue" (i.e it takes the OP +the top level comments, not the pull request review comments)